recipes/sail/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.9.0":
+    url: "https://github.com/HappySeaFox/sail/archive/v0.9.0-pre21.tar.gz"
```

Comment:
so it's not 0.9.0

Reply:
yes, it's a pre-release. I'd like to experiment with Conan before releasing it.